plugins/modules/packaging/language/gem.py

```diff
@@ -135,6 +135,8 @@
 
 import re
 
+from pathlib import PurePosixPath
```

Comment:
Please use `os.path` instead of `pathlib`. The latter is only supported on Python 3.

Reply:
Will do. Thanks for the suggestion.